src/meta/ActiveHostsMan.cpp

```diff
@@ -274,8 +299,8 @@ ErrorOr<nebula::cpp2::ErrorCode, int64_t> LastUpdateTimeMan::get(kvstore::KVStor
   std::string val;
   auto retCode = kv->get(kDefaultSpaceId, kDefaultPartId, key, &val);
   if (retCode != nebula::cpp2::ErrorCode::SUCCEEDED) {
-    LOG(ERROR) << "Get last update time failed, error: "
-               << apache::thrift::util::enumNameSafe(retCode);
+    LOG(INFO) << "Get last update time failed, error: "
```

Comment:
why info?

Reply:
Sorry, I changed it when debug. I'll change it back.